doc/iterations/2024.9/add_support_for_diagram_tool_shortcuts.adoc

```diff
@@ -0,0 +1,69 @@
+= Add support for diagram tool shortcuts
+
+== Problem
+
+Diagram tools are accessible only from the contextual palette of the diagram, a node or an edge
+
+== Key Result
+
+- Specifiers is able to define the shortcut associated to a node or an edge tool in the View model.
+- Users is able to execute the tool by clicking on a diagram/node/edge and using the shortcut
+
+== Solution
+
+*Specifier*
+
+In the view model, there is two new attributes on the edge and node tools.
+* `Tool.shortcutModifiers` This is a multi-valued attribute of `ShortcutModifier`enum.
+Values are `CTRL`, `ALT` and `SHIFT`
+* `Tool.shortcutValue` This is an alphanumeric character
+
+The shortcut is the combinaison of the modifiers and the value.
+
+In details view, a dedicated widget is used to display the shortcut, in a group, with a `MultiSelectWidget` and a `TextfieldWidget`.
```

Comment:
Diagrams can be displayed without any other representations, displaying the shortcut should not involve anything else. Use a button in the DiagramPanel to display the list of shortcuts for example and/or display them the tool sections next to their label